src/service/store/state/state-receipt.ts

```diff
@@ -68,17 +78,18 @@ export class StateReceipt {
     });
     switch (count) {
       case 0:
-        return BulletState.P0;
+        return this._status.bulletState = BulletState.P0;
```

Comment:
why are we returning the result of a setter?

Reply:
ya, it dont make sense, I've removed